### DISCUSSION TITLE
Monarc SaaS - How to restore a backup?

### DISCUSSION THREAD

**Discussion by danje57** [monarc-project/MonarcAppFO#477] [Q&A] [answered]
Dear,

I made a mistake with one of my risk analysis and unfortunately I lost averything in this analysis.

Fortunately, I have a backup. So I could restore it.

However, I don't see any option in the interface which could allow me to restore it.

I think this is because I use the SaaS .

Do you have any suggestion please?

Regards,

**Comment by ruslanbaidan** [comment, marked as answer]
Hello,
Could you clarify the following points:
1. What kind of backup you made (exported analysis, database backup) ?
2. Do you use Monarc on my.monarc.lu ?

**Reply by danje57** [reply]
Thanks for your help,

1) It is an exported backup from the GUI (.bin)
2) Yes I use that I call the SaaS version in my.monarc.lu

Regards,

Jeremy

**Reply by ruslanbaidan** [reply]
Thanks for the answers.
If you exported your analysis from Monarc UI, you can import it any a new or existing analysis.
There is an "Import risk analysis" button just above Export. If it is disabled, then you have only read permissions.
But if you are admin, you can change this.

**Reply by danje57** [reply]
Damned!!! I never seen this option!

Thanks! you save my analysis!!! and my day! :)
